docs/variables.md

```diff
@@ -161,6 +161,16 @@ returns `0`. Please review the Sprig documentation to understand which functions
 | `tasks.<TASKNAME>.outputs.parameters.<NAME>` | Output parameter of any previous task. When the previous task uses 'withItems' or 'withParams', this contains a JSON array of the output parameter values of each invocation |
 | `tasks.<TASKNAME>.outputs.artifacts.<NAME>` | Output artifact of any previous task |
 
+### HTTP Templates
+
+Only available for `successCondition`
+
+| Variable | Description|
+|----------|------------|
+| `statusCode` | Response status code (`int`) |
+| `body` | Response body (`string`) |
+| `headers` | Response headers (`map[string][]string`) |
```

Comment:
Would it be better to scope these, e.g. `http.statusCode` so it's more differentiable when there are other types of templates besides HTTP?

Reply:
Yes, good idea

Reply:
I'm not sure it should be scoped as `http` should probably be like `response`? Would you would to include `request`?

Reply:
Yes, I had made the change using `response`. `request` can be useful too since the template itself is likely to be parametrized. Will include it.

Reply:
Good ideas